src/contexts/Polkawatch/index.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { Configuration, PolkawatchApi } from '@polkawatch/ddp-client';
+import React, { createContext, useContext, useEffect, useState } from 'react';
+import { useApi } from '../Api';
+import type { NetworkName } from '../../types';
+
+/**
+ * This is the Polkawatch API provider, which builds polkawatch API depending on the Chain that is currently
+ * in context. Also returns information about whether there exist decentralization analytics for the Network.
+ */
+
+const POLKAWATCH_API_VERSION = 'v2';
+const POLKAWATCH_NETWORKS = ['polkadot', 'kusama'];
```

Comment:
To maintain conventions these values can go inside a `defaults.ts` file in this same folder. Check out the other contexts to check out the convention.

Reply:
applied